model/pipeline/src/main/proto/beam_runner_api.proto

```diff
@@ -245,6 +246,9 @@ message StandardPTransforms {
     COMBINE_MERGE_ACCUMULATORS = 1 [(beam_urn) = "beam:transform:combine_merge_accumulators:v1"];
     COMBINE_EXTRACT_OUTPUTS = 2 [(beam_urn) = "beam:transform:combine_extract_outputs:v1"];
   }
+  enum SplittableParDoComponents {
+    SPLITTABLE_PROCESS_KEYED = 0 [(beam_urn) = "beam:transform:splittable_process_keyed_elements:v1"];
```

Comment:
Lets add the other two URNs right now as well, what would their payloads be as well?